src/boot/stylesheets/main.scss

```diff
@@ -690,12 +690,21 @@ body {
     				display: block;
     			}
     		}
+
+			p {
+				margin-bottom: 1em;
+			}
     	}
 
     	.wpnc__paragraph:first-of-type {
     		margin-top: 0;
     	}
 
+		.wpnc__body-list {
+			list-style: disc;
+			margin-left: 2em;
```

Comment:
I would do this instead of margin-left

`padding: 0 16px;`

That'll line things up on the sides.

![screen shot 2017-05-23 at 10 48 13 am](https://cloud.githubusercontent.com/assets/618551/26363188/f1ce6a2a-3fa5-11e7-9502-af2266d834c1.png)